app/styles/app.css

```diff
@@ -2,3 +2,15 @@ body {
   min-height: 2000px;
   padding-top: 70px;
 }
+
+.tweet-list {
+  height: 100px;
```

Comment:
Why do you need to hard code the height here?

It would be best if the tweet box automatically took the height of the content. If I remember correctly, you just need to make sure that there is no unnecessary padding at the top of the element.